src/flat.ts

```diff
@@ -54,16 +54,22 @@ async function validateFlatOpts (opts: FlatOptions): Promise<ValidatedFlatOption
  * @returns {Promise} Promise.
  */
 async function buildApplicationPkg (opts: ValidatedFlatOptions, identity: Identity) {
-  const args = ['--component', opts.app, opts.install, '--sign', identity.name, opts.pkg];
+  const componentPkgPath = path.join(path.dirname(opts.app), path.basename(opts.app, '.app') + '-component.pkg');
+  const pkgbuildArgs = ['--install-location', opts.install, '--component', opts.app, componentPkgPath];
```

Comment:
```suggestion
  const pkgbuildArgs = ['--component', opts.app, componentPkgPath];
```

nit: do we still need `--install-location` if we're overwriting it in `--package <pkg-path> <install-path>` below?

Reply:
Since this doesn't seem to affect installation, I'll leave it in.